README.md

```diff
@@ -15,3 +15,32 @@ The following services are part of the data access services (`data-access-servic
 The `data-access-services` sub-project is a multi-module Maven project which includes a library module (`core-banking-lib`) for sharing configuration and code between the data access services.
 
 TODO: Once the data schema and names have been decided this list needs to be updated. Currently, it only includes the transaction service for showing how the multi-module Maven project is set up. Therefore, when adding new services most of the `pom.xml` of the transaction service can be taken over.
+
+# Development
+
+This section describes how to get started with development for this repository.
+
+## Required software
+
+- Git
+- JDK 11
+- TBD
```

Comment:
What exactly is to be defined? Or just a typo?

Reply:
to add other required sw (e.g. for JS development, Docker) later